androidHyperskillApp/src/main/res/layout/fragment_home.xml

```diff
@@ -47,6 +59,27 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="20dp"
+                android:layout_marginHorizontal="20dp"
+                />
+
+            <TextView
+                android:id="@+id/homeTopicsToDiscoverNextTitle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@string/home_topics_to_discover_next_title"
+                android:textAppearance="@style/TextAppearance.AppCompat.Body1"
+                android:textColor="@color/color_on_surface_alpha_60"
+                android:layout_marginTop="46dp"
+                android:layout_marginHorizontal="20dp"
+                />
+
+            <androidx.recyclerview.widget.RecyclerView
+                android:id="@+id/homeTopicsToDiscoverNextRecycler"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginTop="9dp"
+                tools:itemCount="1"
+                tools:listitem="@layout/item_topic"
```

Comment:
For "Learn next topic" on the Homescreen we also need to display the right accessory button, please see the screenshot below from Figma

<img width="733" alt="Screenshot 2023-02-03 at 12 23 26 PM" src="https://user-images.githubusercontent.com/8586063/216519469-e008fc15-1d92-41d0-b318-cc30225b1a9a.png">